src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,153 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
```

Comment:
[Private methods and classes should be after public ones in your class.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/collections/linked-list.html#private-methods-and-classes-should-be-after-public-ones-in-your-class)